index.html

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Project 2: The News Site</title>
+    <link rel="stylesheet" href="./sytles/mobile-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/tablet-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/destokp-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/slideshow-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/slideshow-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/overlay.css" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel='stylesheet' type='text/css'>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+
+
+</head>
+
+<body>
+    <header>
+        <div class="title">
+            <h1>The Ladies Gamer</h1>
+        </div>
+    </header>
+    <hr>
```

Comment:
Check your VSC settings, 2 spaces is enough as indentation :)
![Skärmavbild 2021-02-02 kl  16 28 45](https://user-images.githubusercontent.com/51529201/106622292-b9560980-6573-11eb-961c-81515ee06ac1.png)

Reply:
Thank!!! it's done! in project 3 I have the same issue